.github/workflows/task-stale.yml

```
@@ -19,3 +19,4 @@ jobs:
             stale-pr-label: "stale"
             stale-pr-message: "This pull request is stale because it has been open for 60 days with no activity."
             close-pr-message: "This pull request was closed because it has been inactive for 7 days since being marked as stale."
+            "operations-per-run": 1000
```

shouldn't it be without quotes? 
```suggestion
            operations-per-run: 1000
```